input/system/rds/logs.go

```diff
@@ -88,75 +65,43 @@ func DownloadLogFiles(server *state.Server, logger *util.Logger) (state.Persiste
 			lastMarker = &prevMarker
 		}
 
-		var logFile state.LogFile
-		logFile.UUID = uuid.NewV4()
-		logFile.TmpFile, err = ioutil.TempFile("", "")
+		tmpFile, err := ioutil.TempFile("", "")
 		if err != nil {
 			err = fmt.Errorf("Error allocating tempfile for logs: %s", err)
 			goto ErrorCleanup
 		}
-		logFile.OriginalName = *rdsLogFile.LogFileName
 
 		for {
-			var resp *rds.DownloadDBLogFilePortionOutput
-			resp, err = rdsSvc.DownloadDBLogFilePortion(&rds.DownloadDBLogFilePortionInput{
-				DBInstanceIdentifier: &identifier,
-				LogFileName:          rdsLogFile.LogFileName,
-				Marker:               lastMarker, // This is not set for the initial call, so we only get the most recent lines
-			})
-
+			newBytesWritten, newMarker, additionalDataPending, err := downloadRdsLogFilePortion(rdsSvc, tmpFile, logger, &identifier, rdsLogFile.LogFileName, lastMarker)
 			if err != nil {
-				err = fmt.Errorf("Error downloading logs: %s", err)
-				logFile.Cleanup()
+				tmpFile.Close()
+				os.Remove(tmpFile.Name())
```

Comment:
Same question as the other `Close` and `Remove` comments--just making sure we're ignoring them intentionally.